tests/kernel/protection/README.rst

```diff
@@ -0,0 +1,84 @@
+.. protection_tests:
```

Comment:
Sorry, I missed that it also needs a leading underscore: `.. _protection_tests:`